# Profiling vector integration sites at single-cell level using scATAC-seq or scMultiome data

Here are the steps for profiling VIS from 10X scATAC-seq/scMultiome data. The main tool we used is EpiVIA (https://github.com/wangwl/epiVIA). The starting point is the fastq files generated by 10X scATAC-seq data. There are 4 fq files: I1, R1, R2 and R3. I1 is the 8 bp sample barcode, R1 is the forward read, R2 is the 16 bp 10x feature barcode and R3 is the reverse read. 

As prerequisites, files from multiple lanes should be merged. Qualtiy control was performed for R1 and R3 (fastqc), reads should be trimmed in case adapters are found. More important, build a bwa index for a modified human genome. The provirus sequence (see the file provirus_sequence) is appended as an additional chromosome.


```bash
idx=/research/projects/yu3grp/IO_JY/yu3grp/LVXSCID/patients_scATACseq/bwa_index/hg19withpv/hg19wpvidx ##location of the bwa index
input1=P1_scMulti_ATAC_S1_R1.fastq.gz
input2=P1_scMulti_ATAC_S1_R3.fastq.gz
bam_file=P1_scMulti_ATAC_S1_pe.bam
bwa mem -t 4 $idx $input1 $input2 | samtools view -bS - > $bam_file
```

